test/renderer_test.rb

```diff
@@ -15,6 +15,34 @@ class RendererTest < NicePartials::Test
     assert_css("img") { assert_equal "https://example.com/image.jpg", _1["src"] }
   end
 
+  test "render with options from call site" do
+    render inline: <<~ERB
+      <%= render "columns" do |partial| %>
+        <%= partial.left "The Left", class: "left-column" %>
+        <%= partial.right "The Right", class: "right-column" %>
+      <% end %>
+    ERB
```

Comment:
```suggestion
    render "columns" do |partial|
      partial.left "The Left", class: "left-column"
      partial.right "The Right", class: "right-column"
    end
```

I think we can just render inline (lol) like line 5 does?